src/resources/raids/policestation.ts

```diff
@@ -89,8 +89,8 @@ export const station: Location = {
 		},
 		{
 			type: 'LootChannel',
-			name: 'dome-depot',
-			display: 'Dome Depot',
+			name: 'Uniform Area',
```

Comment:
names must follow discord channel naming rules (lowercase and use - instead of spaces)
```suggestion
			name: 'uniform-area',
```